File: a_easy/11_ab_check/ab_check.rb
def ab_check(str)
  bools = []

  for n in 0..str.length-1 do
    if str[n] == 'a'
      if str[n+4] == 'b'
        bools << 'true'
      else
        bools << 'false'
      end
    end
  end

  if bools.include? 'true'
    true
  else
    false
  end
end

File: a_easy/11_ab_check/ab_check.js
function abCheck(str) {
  var array = str.split("");
  var bools = [];

  for (i = 0; i < array.length; i++) {
    if (array[i] === "a") {
      if (array[i + 4] === "b") {
        bools.push(true);
      } else {
        bools.push(false);
      }
    } else if (array[i] === "b") {
      if (array[i + 4] === "a") {
        bools.push(true);
      } else {
        bools.push(false);
      }
    }
  }

  return bools.includes(true);
}
